### DISCUSSION TITLE
Wrong time is shown

### DISCUSSION THREAD

**Discussion by rares-m** [home-assistant/frontend#10689] [Other feature requests] [answered]
When profile is set to use 24 hour format, the history graph is showing 24 instead of 00.
![Screenshot_20211122-211315__01](https://user-images.githubusercontent.com/76651893/142962403-884c3fe9-a22c-4bb9-bc25-aabfb8b30737.jpg)
![Screenshot_20211122-211218__01](https://user-images.githubusercontent.com/76651893/142962405-ae39426c-6cef-4655-b52a-3323e3080054.jpg)

**Comment by spacegaier** [comment, marked as answer]
It's a known Chrome browser issue and is being worked on: #9379

**Comment by rares-m** [comment]
Thanks!
